x-pack/plugins/security_solution/public/users/pages/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+import { USERS_PATH } from '../../../common/constants';
+import { UsersTableType } from '../store/model';
+import { Users } from './users';
+import { UsersDetails } from './details';
+
+const usersTabPath = `${USERS_PATH}/:tabName(${UsersTableType.allUsers})`;
+
+import { usersDetailsPagePath } from './types';
+
+const usersDetailsTabPath = `${usersDetailsPagePath}/:tabName(${UsersTableType.allUsers})`;
+
+export const UsersContainer = React.memo(() => (
+  <Switch>
+    <Route
+      exact
+      strict
+      path={USERS_PATH}
+      render={({ location: { search = '' } }) => (
+        <Redirect to={{ pathname: `${USERS_PATH}/${UsersTableType.allUsers}`, search }} />
+      )}
+    />
```

Comment:
I think that "app/security/users/" won't match this one, note the final "/".
Maybe we could move this route to the end, removing the `exact` and `strict` props, so anything that doesn't match any other thing will be redirected to the main Users page

Reply:
Good catch! 👍 
I am also applying this fix to the Hosts page!